tools/db_bench_tool.cc

```diff
@@ -896,6 +896,20 @@ DEFINE_int64(max_num_range_tombstones, 0,
 DEFINE_bool(expand_range_tombstones, false,
             "Expand range tombstone into sequential regular tombstones.");
 
+DEFINE_int32(num_overwrites, -1,
+             "Number of overwrites in UpdateRandom benchmark. This "
+             "number is used in combination with num to create a ratio r. "
```

Comment:
Is `r` = `num_o` / `num` or `r` = `num_o` / `(num + num_o)`? That's not clear at all.

Also, I commonly omit --num and use --duration. It's not clear what this parameter means in that case without looking up the default for num. Can we just make it a double for overwrite_probability?

Reply:
Fixed